src/faq.md

```diff
@@ -79,6 +79,55 @@ outputs:
       outputEval: ${self[0].basename=inputs.otu_table_name; return self;}
 ```
 
+## Referencing a local script
+
+We are going to describe two methods of doing this:
+
+The first method involves [adding the folder containing your scripts to the `PATH` environment variable](https://stackoverflow.com/q/8779951/20123733). This allows you to run the shell script without using `sh` or `bash` commands
+
```

Comment:
Missing period in sentence above, and unnecessary two blank lines (I think... I am using GH mobile but I think a single blank line is OK)